src/BlockTypes/ProductGalleryLargeImage.php

```diff
@@ -69,28 +71,23 @@ protected function render( $attributes, $content, $block ) {
 		$processor->remove_class( 'wp-block-woocommerce-product-gallery-large-image' );
 		$content = $processor->get_updated_html();
 
-		$image_html = wp_get_attachment_image(
-			$product->get_image_id(),
-			'full',
-			false,
-			array(
-				'class' => 'wc-block-woocommerce-product-gallery-large-image__image',
-			)
-		);
+		[  $visible_main_image, $main_images ] = $this->get_main_images_html( $block->context, $post_id );
```

Comment:
There seems to be an extra space here.
```suggestion
		[ $visible_main_image, $main_images ] = $this->get_main_images_html( $block->context, $post_id );
```